### PR TITLE
Add column rollup_map.raw_soql

### PR DESCRIPTION
**Pull request by chitang**

### CODE REVIEW

File: coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20220415-add-column-rollup-raw-soql.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="Chi" id="20220415-add-column-rollup-raw-soql">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="rollup_map" columnName="raw_soql"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="rollup_map">
+            <column name="raw_soql" type="varchar"/>
```
Comment (saraid) [inline coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20220415-add-column-rollup-raw-soql.xml:13]: Vaguely surprised you're using `varchar` here. Or does that no longer have a reachable max length?

**Comment by chitang** [comment]
Pretty insignificant and for convenience at very little cost.
user_soql might be a better name

**Review by saraid — APPROVED** [review]
I don't know enough about this to have a problem with it.

**Comment by chitang** [comment]
The field rollup_map.soql is a transformed soql with internal column ids which is not for human readability.  In theory, we have code to convert this soql back and forth.  But it is a difficult process.  Bugs can also get in the way.  It would be nice to save a copy of the original soql given by the user.

File: coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20220415-add-column-rollup-raw-soql.xml (hunk 1 above)
Comment (chitang) [reply coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20220415-add-column-rollup-raw-soql.xml:13]: This field mirrors rollup_map.soql which is varchar.

**Pull request merged**